### DISCUSSION TITLE
Mantine css is not working with react js

### DISCUSSION THREAD

**Discussion by RefreshIdeas** [mantinedev/mantine#6020] [Q&A]
I dont know why mantine css is not working with react js(no typescript)

npm - npm install @mantine/core @mantine/hooks
`<MantineProvider withGlobalStyles={true} withNormalizeCSS={true}> <BrowserRouter> <AllRoutes /> </BrowserRouter> </MantineProvider>`

**Comment by rtivital** [comment]
https://mantine.dev/getting-started/
